compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/MouseWheelScrollable.kt

```diff
@@ -229,13 +230,13 @@ private class AnimatedMouseWheelScrollPhysics(
         var requiredAnimation = true
         var lastValue = 0f
         val anim = AnimationState(0f)
-        while (requiredAnimation) {
+        while (requiredAnimation && coroutineScope.isActive) {
```

Comment:
```suggestion
        while (requiredAnimation && coroutineContext.isActive) {
```

We can wrap this suspend function into another `coroutineScope { }` in the future. Better to be safe.